build.gradle

```diff
@@ -203,7 +203,8 @@ sonarqube {
     property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/config/**," +
       "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/Application.java" +
       "src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/**," +
-      "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/clients/**"
+      "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/clients/**" +
+      "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/entities/**"
```

Comment:
Any particular reason on why we should exclude the entities from sonar I think you've covered most of them using the pojo tester

Reply:
not required. Will delete